sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitEditForm.java

```diff
@@ -113,7 +113,8 @@ protected void addFields() {
 
 		TextField textFieldPhone = new TextField(I18nProperties.getCaption(Captions.contactPersonPhoneNumber));
 		textFieldPhone.setWidth(100, Unit.PERCENTAGE);
-		textFieldPhone.setValue(this.person.getPhone());
+		String phoneNumber =  (this.person.getPhone() == null) ? "" : this.person.getPhone();
```

Comment:
You should use "setNullRepresentation" instead of doing a manual null check.

Reply:
@MateStrysewske, my CI/CD(Travis) build is failing after merging with development and I think I am not the only one experiencing this problem. As can be seen from the build log below. A test in sormas-backend which has nothing to do with my change is failing.

<img width="1443" alt="Screen Shot 2020-09-20 at 18 01 20" src="https://user-images.githubusercontent.com/58990931/93717550-75d3a380-fb6e-11ea-9e6c-115969da3c95.png">

Reply:
Yeah, that's fine, it's a result of the latest hotfix.